### ISSUE TITLE
Where are the ios and android folders?

### ISSUE THREAD

**Issue by yaronlevi** [exponentjs/xde#62]
I've just created a new project with xde, but looking at the folders created, I can't seem to find the xcode projects files. (I need to change them to add some external libs. for example, change the plist file).

**Comment by jesseruder** [comment]
There isn't iOS and Android code with Exponent. See https://docs.getexponent.com/versions/latest/introduction/faq.html#what-is-the-difference-between-exponent-and-react-native

**Comment by yaronlevi** [comment]
But many external libs require me to manually change files in those folders. For example [react-native-sound](https://github.com/zmxv/react-native-sound), requires me to make some changes to the xcode's project file. How can I solve this in Exponent?

**Comment by brentvatne** [comment]
@yaronlevi - see this post: http://stackoverflow.com/questions/39170622/what-is-the-difference-between-exponent-and-react-native/39170715#39170715

**Comment by yaronlevi** [comment]
I see now. So let me rephrase: How can I make a **sound** with exponent + react native.
I could not find any sdk in exponent for that, and as you mentioned I am locked in and can't use any 3rd party libs that require changing the native code. I love Exponent and want to continue using it, how can I pass this hurdle?

**Comment by terribleben** [comment]
Hey @yaronlevi thanks for asking - we are working on a sound API for one of the next SDKs (though probably not SDK 11) because you are not the only person who has asked for sound.

The best solution in the meantime is actually to use our video API to play a sound: https://docs.getexponent.com/versions/v10.0.0/sdk/video.html

**Comment by yaronlevi** [comment]
Great thanks. Must say XDE is just pure magic. I've created a prototype of a real time map collaboration app with airbnb's react-native-maps + PubNub, then used Exponent to demo it to a customer on 10 diffrent devices.
All this, in **one hour**.
